src/components/Tiles/TilesGrid.scss

```diff
@@ -0,0 +1,12 @@
+.grid-container {
+    display: grid;
+
+    grid-template-columns: repeat(auto-fit, 380px);
+    justify-content: center;
+}
+
+@include for-phone-and-tablet {
```

Comment:
If you want shorter and more scss like syntax you can go with

`.grid-container {
    display: grid;

    grid-template-columns: repeat(auto-fit, 380px);
    justify-content: center;
    @include for-phone-and-tablet {
        grid-template-columns: repeat(auto-fit, 140px);
    }
}


`

Reply:
oh yeah I do want that